samples/excel/50-workbook/create-workbook.yaml

```diff
@@ -0,0 +1,86 @@
+order: 2
+id: excel-workbook-create-workbook
+name: Create workbook
+description: Sample for Excel
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.8
+script:
+    content: |
+        $("#file").change(() => tryCatch(createWorkbook));
+
+        async function createWorkbook() {
+            const myFile = document.getElementById("file");
+            const reader = new FileReader();
+
+            reader.onload = ((theFile) => {
+                return ((event) => {
+                    Excel.run(context => {
+                        // strip off the metadata before the base64-encoded string
+                        const startIndex = event.target.result.indexOf("base64,");
+                        const mybase64 = event.target.result.substr(startIndex + 7);
+
+                        Excel.createWorkbook(mybase64);
+                        return context.sync();
+                    });
+                });
+            })(myFile.files[0]);
+
+            // read in the file as a data URL so we can parse the base64-encoded string
+            reader.readAsDataURL(myFile.files[0]);
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            } catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
```

Comment:
I don't understand what this sample is supposed to do. the only UI is a browse control. I browse to an Excel file and select it. But then nothing happens. Is something supposed to happen?

Reply:
It's supposed to open a copy of the workbook in a new instance. What environment are you running this snippet in?

Reply:
@AlexJerabek Office 365 1812 11113.20002 Click to Run. I've tried both prod and beta Script Lab.